ocp3/product.yml

```diff
@@ -2,7 +2,7 @@ product: ocp3
 full_name: Red Hat OpenShift Container Platform 3
 type: platform
 
-benchmark_root: "../applications"
+benchmark_root: ["../applications","../linux_os/guide/system"]
```

Comment:
I envision a problem that we will have to spend a lot of time with "prodtype" elements in "rule.yml"s. Currently, this adds rules eg. "Disable WIFI Network Connection Creation in GNOME" to the OCP content, which is  a rule that doesn't make sense in OpenShift context.

Reply:
That is exactly my concern in #4908. I have some doubts about this multiroot thing being actually useful.

Reply:
There are more mutiroot benchmarks coming, so it is useful, yes.